### DISCUSSION TITLE
Duplicate faq page in docs

### DISCUSSION THREAD

**Discussion by Mister-Hope** [Mereithhh/vanblog#186] [Q&A] [answered]
I found faq is duplicated in docs, I am planning to merge them and import `vuepress-plugin-redirect` to automatically redirect the other page to the remain one.

So shall I remain the page under guide, or the page under ref? @Mereithhh

**Comment by Mister-Hope** [comment, marked as answer]
I also found more duplicate page, so I will wait until you give me some feedback.

Also the docs is too colloquialism instead of professional

**Reply by Mereithhh** [reply]
@Mister-Hope Remain the page under guide is fine. You can decide which one to keep for other duplicate pages.

Forgive my poor writing skill 😂, Maybe I will rewrite them sometimes.

**Reply by Mister-Hope** [reply]
If you don't mind, I can rewrite some of them for you

**Reply by Mereithhh** [reply]
Never mind. Just do it !

**Reply by Mereithhh** [reply]
No problem !
